prime-router/src/main/kotlin/azure/SubmissionsFacade.kt

```diff
@@ -34,8 +34,8 @@ class SubmissionsFacade(
     // Leaving separate from FindSubmissions to encapsulate json serialization
```

Comment:
Why is this comment not part of the kdoc above it? It looks like it would be good to know the differences between the two find submissions methods.

Reply:
Not sure the comment helps.  Removed.